common/datavalues/src/types/data_type.rs

```diff
@@ -125,7 +126,15 @@ pub trait DataType: std::fmt::Debug + Sync + Send + DynClone {
 
     fn create_mutable(&self, capacity: usize) -> Box<dyn MutableColumn>;
     fn create_serializer(&self) -> TypeSerializerImpl;
+    /// work only for timestamp serializer
+    fn create_serializer_with_tz(&self, _tz: Tz) -> TypeSerializerImpl {
+        unimplemented!()
+    }
     fn create_deserializer(&self, capacity: usize) -> TypeDeserializerImpl;
+    /// work only for timestamp deserializer
+    fn create_deserializer_with_tz(&self, _capacity: usize, _tz: Tz) -> TypeDeserializerImpl {
```

Comment:
Can we pass tz into `FormatSettings` ? Thus we don't need this function.

Reply:
we can not pass tz into `FormatSettings` because `FormatSettings` has derived `Serialize` and `Deserialize` trait and `chrono_tz::Tz` do not implement the trait.

and another way is to pass `FormatSettings` when we call `deserialize_xxxx`, and then we'll need to parse `tz: vec<u8>` to `tz: Tz` every time we call `deserialize_xxxx`